buildsystem/dependencies.gradle

```diff
@@ -0,0 +1,77 @@
+allprojects {
```

Comment:
it is outdated now, the modern way is create `buildSrc` and `kotlinDsl` refer project `iosched` to know about it. and refactor later